guides/module.txt

```diff
@@ -0,0 +1,23 @@
+TOPIC
+    module
+
+DESCRIPTION
+    Modules are shareable, reusable packages of Puppet content. They can include
+    tasks, plans, functions, and other types of content that you can use in your
+    project. You can download and install modules to your project from the
+    Puppet Forge or write your own modules. Bolt also ships with several helpful
+    modules pre-installed that are available to all of your projects.
+
+    When Bolt runs a command, it automatically loads modules from specific
```

Comment:
It might be useful to just have a separate modulepath guide. As a new person, it seems like it'd be confusing to think about both the `modules` configured in the project and the modulepath. Something like "to learn more about how modules are loaded, run 'bolt guide modulepath'" and then getting more in depth about the default path and what special paths are (i.e. `modules` is for custom modules, `.modules` is where modules get installed, etc.).

Reply:
Oh that's a great idea!